### ISSUE TITLE
tsc build errors

### ISSUE THREAD

**Issue by kodemill** [HarmonicLabs/plu-ts#3]
Hey, I'm trying to build the project, but run into some typescript compiler issues (on both `main` and `dev` branches):

```bash
npm run build

[...]

Found 149 errors in 16 files.

Errors  Files
     3  src/onchain/pluts/PType.ts:15
    12  src/onchain/pluts/PTypes/PData/conversion/index.ts:35
     3  src/onchain/pluts/PTypes/PFn/PFn.ts:9
     7  src/onchain/pluts/PTypes/PFn/PLam.ts:37
     2  src/onchain/pluts/PTypes/PStruct/pstruct.ts:49
     3  src/onchain/pluts/Syntax/experimental/pconst.ts:79
     7  src/onchain/pluts/Syntax/pappArg.ts:33
    16  src/onchain/pluts/Syntax/syntax.ts:42
    60  src/onchain/pluts/Term/Type/base.ts:70
     1  src/onchain/pluts/Term/index.ts:67
     7  src/onchain/pluts/stdlib/Builtins.ts:89
     2  src/onchain/pluts/stdlib/UtilityTerms/TermAlias.ts:10
     7  src/onchain/pluts/stdlib/UtilityTerms/TermStruct.ts:15
     9  src/onchain/pluts/stdlib/UtilityTerms/addUtilityForType.ts:33
     8  src/onchain/pluts/stdlib/UtilityTerms/types.ts:30
     2  src/utils/ts/TyLists.ts:3
```

*OS*: linux
*Related versions*:
```bash
node -v
v16.18.1
```
```bash
npx tsc -v                      
Version 4.6.3
```

I've forked the repo and created a Github workflow to test this on other machines:
https://github.com/kodemill/plu-ts/actions/runs/3554670173/jobs/5970909622

**Comment by HarmonicPool** [comment]
Thank you for opening an issue;

it was my fault; bad `tsconfig` and `build` script setup; it should be fixed with version 0.1.5

please update your `package.json` to
```json
"dependencies": {
    "@harmoniclabs/plu-ts": "^0.1.5"
}
```
and then run `npm install`;

Let me know if this fixes the issue;

documentation is being updated accordingly;

**Comment by kodemill** [comment]
Nice, that did the trick: https://github.com/kodemill/plu-ts/actions/runs/3573239349/jobs/6007067166
Thanks for the quick fix, and for this awesome project!
